crates/stackable-telemetry/src/tracing.rs

```diff
@@ -41,6 +45,8 @@ pub enum Error {
 ///
 /// #[tokio::main]
 /// async fn main() -> Result<(), Error> {
+///     // IMPORTANT: Name the guard variable appropriately, do not just use
```

Comment:
suggestion: Let's include this notice in the doc comment directly (maybe in addition to this note).